crates/astria-conductor/src/data_availability/block_verifier.rs

```diff
@@ -423,32 +409,18 @@ mod test {
         let (validator_set, proposer_address, commit) =
             make_test_validator_set_and_commit(height, header.chain_id.clone());
         header.proposer_address = proposer_address;
-        let block_hash = header.hash();
 
-        let sequencer_namespace_data = RawSequencerNamespaceData {
-            block_hash,
+        let sequencer_blob = UncheckedCelestiaSequencerBlob {
             header,
-            rollup_chain_ids: vec![chain_id],
-            action_tree_root,
-            action_tree_root_inclusion_proof,
-            chain_ids_commitment,
-            chain_ids_commitment_inclusion_proof,
+            rollup_ids: vec![rollup_id],
+            rollup_transactions_root,
+            rollup_transactions_proof,
+            rollup_ids_proof,
         }
-        .try_into_verified()
+        .try_into_celestia_sequencer_blob()
         .unwrap();
 
-        let rollup_namespace_data = RollupNamespaceData {
-            block_hash,
-            chain_id,
-            rollup_txs: vec![test_tx],
-            inclusion_proof: action_tree.construct_proof(0).unwrap(),
-        };
-
-        validate_sequencer_namespace_data(&validator_set, &commit, &sequencer_namespace_data)
-            .unwrap();
-        rollup_namespace_data
```

Comment:
This check no longer made sense because the method to fetch the blobs in `astria-celestia-client` now also ensures that only blobs that are related to the sequencer blob are returned.